src/Router/RabbitMq/Adapter.php

```diff
@@ -25,16 +28,20 @@ final class Adapter implements LoggerAwareInterface
      * @var int|null
      */
     private $timeout;
-
     /**
      * @var bool
      */
     private $rejectToBottom;
+    /**
+     * @var ParameterBagInterface
+     */
+    private $parameterBag;
 
-    public function __construct()
+    public function __construct(ParameterBagInterface $parameterBag)
```

Comment:
It's preferable to use a setter like the timeout, this way is easier to handle default value instead of the magic of parameters declaration, and simplify testing

Reply:
I don't know, maybe i should a private "delayedExchange" property and add a setter. But i think i should instanciate this property using the parameterBag. It makes the property configurable from the console or from the parameters file

Reply:
Absolutly but injecting ParameterBag is not recommanded, in this case it should have been `string $delayedQueue` with a DI configuration

Reply:
I'll just remove the parameter part and let the console option with a default value